packages/@react-types/progress/src/index.d.ts

```diff
@@ -68,6 +68,8 @@ export interface SpectrumProgressCircleProps extends AriaProgressCircleProps, St
    * @default 'M'
    */
   size?: 'S' | 'M' | 'L',
+  /** The static color style to apply. Useful when the button appears over a color background. */
+  staticColor?: 'white' | 'black',
   /** The [visual style](https://spectrum.adobe.com/page/progress-circle/#Over-background-variant) of the ProgressCircle. */
   variant?: 'overBackground'
```

Comment:
Should we deprecate `variant` now that `staticColor="white"` is the same thing? We'd have to keep it in the types/implementation for backward compatibility, but we could add `@deprecated` and remove the example from the docs.

Reply:
+1, @iamwillpowell I think we agreed on doing something similar in otter time, don't quite remember the exact approach (I should've jotted something down).